src/cockpit/channels/filesystem.py

```diff
@@ -114,7 +115,7 @@ class FsReadChannel(GeneratorChannel):
 
     def do_yield_data(self, options: JsonObject) -> GeneratorChannel.DataGenerator:
         path = get_str(options, 'path')
-        binary = get_str(options, 'binary', None)
+        binary = get_enum(options, 'binary', ['raw'], None) is not None
```

Comment:
So the binary option can be bool or 'raw' that is rather confusing. Maybe an idea we deprecate the `raw` option?

Reply:
It can never be bool.  This is a common source of confusion.  It must always be either absent or the literal string `"raw"`

Reply:
Oh it's the filesystem channel *sigh*. Well confusing regardless :)